docs/section19/19-10.md

```diff
@@ -0,0 +1,73 @@
+3. Systems that serve computer rooms complying with Section G3.1.2.6.1.
+
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE/UNDETERMINED       
+**Appendix G Section:** G3.1.2.6 including exceptions 2 and 3.      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. Applies to buildings NOT in climate zones 0A, 0B, 1A, 1B, 2A, 3A, and 4A.   
+2. Applies to baseline HVAC system types 3 through 8, and 11, 12, and 13.   
+ 
+
+**Function Calls:**  
+1. get_baseline_system_types  
+2. get_HVAC_systems_primarily_serving_comp_rooms  
+3. get_hvac_zone_list_w_area  
+4. Get_list_hvac_systems_associated_with_zone  
+
+## Rule Logic:   
+**Applicability Check 1 -  Check if Climate Zone Requires Economizer**  
+- Get climate zone: `climate_zone = ASHRAE229.weather.climate_zone` 
+- If climate zone is not 0A, 0B, 1A, 1B, 2A, 3A, and 4A then carry on, otherwise outcome for all HVAC systems NOT_APPLICABLE: `if climate_zone Not in ["CZ0A", "CZ0B", "CZ1A", "CZ1B", "CZ2A", "CZ3A", "CZ4A"]:`  
+    **Applicability Check 2 - Check if there are any Applicable Baseline System Types in the Project at All**   
+    - Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+    - Check if B-RMR is modeled with at least one air-side system type 3 through 8, or 11, 12, and 13, if yes then carry on, if no then NOT_APPLICABLE for the project: `for sys_type in ["SYS-3", "SYS-3a", "SYS-3b", "SYS-3c", "SYS-4", "SYS-5", "SYS-5b", "SYS-6", "SYS-6b","SYS-7", "SYS-7a", "SYS-7b", "SYS-7c","SYS-8", "SYS-8a", "SYS-8b", "SYS-8c","SYS-11.1", "SYS-11.1a", "SYS-11b", "SYS-11c","SYS-11.2","SYS-11.2a","SYS-12", "SYS-12a", "SYS-12b", "SYS-12c","SYS-13", "SYS-13a"]):`
+        - Check if values are not none for system type key in the dictionary, if it does not equal false then the values are not none and there are applicable system types in the project: `if bool(baseline_system_types_dict.get(sys_type)) != False: is_there_applicable_sys_type = TRUE`  
+    - Check if is_there_applicable_sys_type == True which means there are applicable system types in the project, if yes then carry on, if no then NOT_APPLICABLE for the project: `if  is_there_applicable_sys_type == True:`  
+        This segment creates a list of baseline hvac system IDs in which G3.1.2.6 exception #2 is likely applicable, the exceptions states "where the use of outdoor air for cooling will affect supermarket open refrigerated case-work systems. This exception shall only be used if the system in the proposed design does not use an economizer. If the exception is used, an economizer shall not be included in the baseline building design."
+        - For each building in the B_RMI: ``for bldg in B_RMI...Building:`  
+            - Check of the building has refrigerated cases: `if bldg.refrigerated_cases != Null:`  
```

Comment:
I left as is since the outcome is undetermined anyway, do you think we should change the outcome to undetermined for all retail and just remove this condition?